README.md

```diff
@@ -0,0 +1,70 @@
+# Rancher Labs Documentation
+This readme file includes information pertaining to the documentation project for both Rancher and Rancher OS.
+## Rancher
+Rancher is an open source project that provides a complete platform for operating Docker in production. It provides infrastructure services such as multi-host networking, global and local load balancing, and volume snapshots. It integrates native Docker management capabilities such as Docker Machine and Docker Swarm. It offers a rich user experience that enables devops admins to operate Docker in production at large scale.
+## Rancher OS
+The smallest, easiest way to run Docker in production at scale. Everything in RancherOS is a container managed by Docker. This includes system services such as udev and rsyslog. RancherOS includes only the bare minimum amount of software needed to run Docker. This keeps the binary download of RancherOS to about 20MB. Everything else can be pulled in dynamically through Docker.
+## Rancher Labs Documentation Website
+Rancher documentation is available at <http://docs.rancher.com/rancher/>
+Rancher OS documentation is available at <http://docs.rancher.com/os/>
+## Contributing to Rancher Labs Documentation Project
+### About Rancher Labs Documentation Site
+Rancher Lab documentation is hosted on Githubpages, and published it online by using Jekyll, an easy blog-aware static website generator. For more details on how to set up Jekyll, we recommend you to read <https://help.github.com/articles/using-jekyll-with-pages/>. If you are using Windows, we strongly advise you to follow the instruction given at <http://jekyllrb.com/docs/windows/>.
```

Comment:
Can you just fix the typo for "Rancher Lab" to "Rancher Labs"? I think it's also GitHub Pages. And grammatically it should just say "published online".